### ISSUE TITLE
PHP Warning:  regarding aerospike.so

### ISSUE THREAD

**Issue by LarryEitel** [LaraDock/laradock#282]
FYI - No biggie. I noticed the following when I was firing up xDebug on php-fpm.

```
$ ./xdebugPhpFpm start
Start xDebug
laradock_php-fpm_1
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/aeros
pike.so' - /usr/local/lib/php/extensions/no-debug-non-zts-20151012/aerospike.so: cannot open shared object file: No such
 file or directory in Unknown on line 0
PHP 7.0.9 (cli) (built: Aug 10 2016 19:45:48) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Xdebug v2.4.1, Copyright (c) 2002-2016, by Derick Rethans
```

**Comment by douglascorrea** [comment]
I have this message:
```
HP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20151012/aerospike.so' - /usr/lib/php/20151012/aerospike.so: cannot open shared object file: No such file or directory in Unknown on line 0
```

All the time I execute a PHP command

**Comment by everaldo** [comment]
Regardeless of the PHP Warning, I noticed that things are working fine.

**Comment by luciano-jr** [comment]
I will look for a solution for this warning

**Comment by david-perez** [comment]
This was fixed by 3a5100097ab0eb9144c385a1e516ac9a54077bdf
